reference/api-definition.md

```diff
@@ -50,3 +50,78 @@ during the bootstrapping process, but before any of your resource definitions
 are loaded. This allows you to refer to the contents of your ApiDefinition from
 your resource definitions. See [Bootstrapping](../bootstrapping/) for more
 information on the various bootstrapping stages.
+
+# Global Information
+
+It is possible to provide some global API information in the `ApiDefinition.define` block with the `info` method. You may define this metadata for all versions of your API, or only for a specific version. All definitions at the global level (i.e. those that do not specify a version) will be inherited by all versions. Any directive defined within a version will overwrite anything inherited, with the exceptions of `base_path` and `base_params`, which will always be enforced when defined globally.
+
+The following directives are supported:
+
+ * `name`
+ * `title`
+ * `description`
+ * `base_path`: Root path prepended to *all* routes.
+ * `base_params`: Default parameters applied to all actions. Used to define any params specified in `base_path`.
+ * `consumes`: List of [handlers](../handlers) the API accepts from clients (defaults to `'json', 'x-www-form-urlencoded'`).
+ * `produces`: List of [handlers](../handlers) the API may use to generate responses (defaults to `'json'`).
+
+
+In addition to the above which may be specified globally or on a version, there is a `version_with` directive that is only applicable on the global version to define what versioning "scheme" your application will use. By default it's set to `[:header, :params]`, meaning Praxis will look for either an  `X-Api-Version` request header *or* an `api_version` query parameter to determine the version of your API to use for processing the request. It is also possible to use a path-based versioning scheme by using `version_with :path`. See section below for details.
+
+There is also an  `endpoint` directive to define your API's endpoint. It's used purely for documentation purposes and will not be used in any routing or route-generation.
```

Comment:
The difference with `version_with` and `enpoint` is that they are *only* valid at the global level.